lib/puppet/ssl/host.rb

```diff
@@ -124,10 +116,6 @@ def generate_certificate_request(options = {})
     true
   end
 
-  def http_client(ssl_context)
```

Comment:
I don't see where http_client was used in our code base.  Is this something that just hasn't been removed yet, or could it be used by modules or other user code?

Reply:
We had plans on using this client, but it didn't happen, and it's just dead code.